pkg/apis/velero/v1/labels_annotations.go

```diff
@@ -50,4 +50,16 @@ const (
 	// ResticVolumeNamespaceLabel is the label key used to identify which
 	// namespace a restic repository stores pod volume backups for.
 	ResticVolumeNamespaceLabel = "velero.io/volume-namespace"
+
+	// SourceClusterK8sVersionLabel is the label key used to identify the k8s
+	// git version of the backup , i.e. v1.16.4
+	SourceClusterK8sVersionLabel = "velero.io/source-cluster-k8s-version"
```

Comment:
What do you think of renaming this to `SourceClusterK8sGitVersionLabel`? I think it would disambiguate what specific version this is for for those who don't recall the precise name of the 3 variables.

Reply:
Yes, sure,  I can do that, will tweak the code and amend the commit.

Reply:
I added the suggestions on the last commit.